ColumnReinforcement/SingleTie.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+# ***************************************************************************
+# *                                                                         *
+# *   Copyright (c) 2019 - Suraj <dadralj18@gmail.com>             *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "Single Tie Reinforcement"
+__author__ = "Suraj"
+__url__ = "https://www.freecadweb.org"
+
+import FreeCADGui
+import sys
+
+sys.path.append("../")
```

Comment:
Remove this line.

Reply:
Removed line containing sys in commit https://github.com/amrit3701/FreeCAD-Reinforcement/pull/15/commits/be59ff1a72205f9e6b3b1f738c886601327a5312